static/app/views/codecov/tokens/tokensWrapper.tsx

```diff
@@ -1,31 +1,36 @@
 import {Outlet} from 'react-router-dom';
 import styled from '@emotion/styled';
 
-import {FeatureBadge} from 'sentry/components/core/badge/featureBadge';
+import CodecovQueryParamsProvider from 'sentry/components/codecov/container/codecovParamsProvider';
 import * as Layout from 'sentry/components/layouts/thirds';
+import QuestionTooltip from 'sentry/components/questionTooltip';
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import useOrganization from 'sentry/utils/useOrganization';
 import {TOKENS_PAGE_TITLE} from 'sentry/views/codecov/settings';
 
 export default function TokensPageWrapper() {
   const organization = useOrganization();
 
+  const tooltip = 'Manage your upload tokens that are created in Sentry Prevent.';
```

Comment:
```suggestion
  const tooltip = t('Manage your upload tokens that are created in Sentry Prevent.');
```

Wrap the rendered string in the `t` function for language support.

Reply:
Ahh yeah thanks, added inhttps://github.com/getsentry/sentry/pull/94248/commits/704c236d1772ebd8e014ead9863eb8d504c6d8e5